apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -1060,6 +1053,30 @@ export const platform: NavMenuConstant = {
         { name: 'Production Checklist', url: '/guides/platform/going-into-prod' },
       ],
     },
+    {
+      name: 'OAuth Apps (Beta)',
+      url: undefined,
+      items: [
+        {
+          name: 'Publish an OAuth App',
+          url: '/guides/platform/oauth-apps/publish-an-oauth-app',
+        },
+        {
+          name: 'Authorize an OAuth App',
+          url: '/guides/platform/oauth-apps/authorize-an-oauth-app',
+        },
+      ],
+    },
+    {
+      name: 'Integrations',
+      url: undefined,
+      items: [
+        {
+          name: 'Supabase Marketplace',
+          url: '/guides/platform/marketplace',
+        },
+      ],
+    },
```

Comment:
```suggestion
    {
      name: 'Integrations',
      url: undefined,
      items: [
        {
          name: 'Integrations Marketplace',
          url: '/guides/platform/marketplace',
        },
        {
          name: 'Publish an OAuth App',
          url: '/guides/platform/oauth-apps/publish-an-oauth-app',
        },
        {
          name: 'Authorize an OAuth App',
          url: '/guides/platform/oauth-apps/authorize-an-oauth-app',
        },
      ],
    },
```

I'd just group all of these into one section - they all "fit" together pretty nicely

Reply:
Fixed.